installer_library.sh

```diff
@@ -173,7 +173,7 @@ install_package_linux() {
 			errormessage=$( $SUDOX $INSTALL_CMD $INSTALL_CMD_ARGS $package > /dev/null 2>&1)
 		else
 			# Install it
-			errormessage=$( $SUDOX $INSTALL_CMD update -qq && $SUDOX $INSTALL_CMD $INSTALL_CMD_ARGS --no-install-recommends -yqq $package)
+			errormessage=$( $SUDOX $INSTALL_CMD update -qq && $SUDOX DEBIAN_FRONTEND=noninteractive $INSTALL_CMD $INSTALL_CMD_ARGS --no-install-recommends -yqq $package)
```

Comment:
export DEBIAN_FRONTEND=noninteractive

Reply:
But then we export it also on freebsd or macos ... makes that really sense? Thats why I added it to the relevant apt-get only commands ....